app/Http/Controllers/API/Schemas/Bioschema/BiochemaController.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace App\Http\Controllers\API\Schemas\Bioschema;
+
+use App\Http\Controllers\Controller;
+use App\Models\Bioschema\BioSchema;
+use App\Models\Dataset;
+use App\Models\Project;
+use App\Models\Study;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Spatie\SchemaOrg\Schema;
+
+class BiochemaController extends Controller
+{
+    /**
+     * Implement Bioschema profiles on nmrXiv project, study, and dataset.
+     */
+    public function conforms()
+    {
+        /**
+         * Create a variable $confromsTo for Bioschemas' study creative work.
+         *
+         * @return object $confromsTo
+         */
+        $creativeWork = Schema::creativeWork();
+        $creativeWork['@id'] = 'https://bioschemas.org/profiles/Study/0.2-DRAFT';
+        $confromsTo = [];
+        $confromsTo['http://purl.org/dc/terms/conformsTo'] = $creativeWork;
+
+        return $confromsTo;
+    }
+
+    public function datasetShort($dataset)
+    {
+        /**
```

Comment:
Comment block should precede the function, not with in the function. This needs to be updated in all the places.